docs/docs/code/python/README.md

```diff
@@ -0,0 +1,279 @@
+
+r = requests.get(url)
+
+# to see the response in your Pipedream step logs:
+print(r.text)
+
+# to see the response status code in your Pipedream step logs:
+print(r.status)
+```
+
+### Making a POST request
+
+```python
+import requests
+
+# This a POST request to this URL will echo back whatever data we send to it
+url = 'https://postman-echo.com/post'
+
+data = {"name": "Bulbasaur"}
+
+r = requests.post(url, data)
+
+# to see the response in your Pipedream step logs:
+print(r.text)
+
+# to see the response status code in your Pipedream step logs:
+print(r.status)
+```
+
+### Using API key authentication
+
+If an particular service requires you to use an API key, you can pass it via the headers of the request.
+
+This proves your identity to the service so you can interact with it:
+
+```python
+import requests
+
+token = 'replace this with your API key'
+
+url = 'https://api.twitter.com/2/users/@pipedream/mentions'
+
+headers { 'Authorization': f"Bearer {token}"}
+r = requests.get(url, headers=headers)
+
+print(r.text)
+```
+
+```note
+Not all APIs use a `Authorization: Bearer <api key>` header format.
+
+Refer to the documentation of the API you're working with for their specific authentication format.
+```
+
+### Sending files
+
+You can send also files within a step.
+
+An example of sending a previously stored file in the workflow's `/tmp` directory: 
+
+```python
+# Retrieving a previously saved file from workflow storage
+files = {'image': ('python-logo.png', open('/tmp/python-logo.png', 'rb')}
+
+r = requests.post(url='https://api.imgur.com/3/image', files=files)
+```
+
+## Sharing data between steps
+
+You can share data between steps in your workflows. Data can be passed forward to the next steps, and it can be used by other steps downstream.
+
+### Using data from another step
+
+You may need to use data from a prior step in your workflow.
+
+For example, when your trigger step accepts data from an incoming HTTP request you may need to send that data to another API, transform that data or store it somewhere.
+
+In this example, we'll pretend this data is coming into our HTTP trigger.
+
+```json
+{
+  "id": 1,
+  "name": "Bulbasaur",
+  "type": "plant"
+}
+```
+
+Every Python step can import the `pipedream.script_helpers` module. This module contains data from other steps in the workflow via `steps` dictionary.
+
+Here's an example of how to get the ID from the Pokemon we received in the trigger.
+
+```python
+from pipedream.script_helpers import (steps, export)
+
+# retrieve the data points from the HTTP request in the initial workflow trigger 
+name = steps["trigger"]["event"]["name"]
+pokemon_type = steps["trigger"]["event"]["type"]
+
+print(f"{pokemon_name} is a {pokemon_type} type Pokemon")
+```
+
+### Sending data downstream to other steps
+
+The trigger steps automatically export their data for other steps to use downstream.
+
+Just pass the data to the `export` module from `pipedream.script_helpers`.
```

Comment:
`export` is a function so I would say:

> Call the `export` function to return data from a step

Reply:
Agreed, "automatically" is not accurate.